das/coordinator_test.go

```diff
@@ -13,19 +13,43 @@ import (
 	"github.com/stretchr/testify/assert"
 )
 
-func TestCoordinator(t *testing.T) {
-	concurrency := 10
-	samplingRange := uint64(10)
-	networkHead := uint64(500)
-	sampleFrom := uint64(genesisHeight)
-	timeoutDelay := 125 * time.Second
+type testParams struct {
+	networkHead  uint64
+	sampleFrom   uint64
+	timeoutDelay time.Duration
+	dasParams    dasingParams
+}
+
+func defaultDasParams() dasingParams {
```

Comment:
why is this defined again?

Reply:
good call, will remove. The original intention of this was `defaultTestDasParams` but we can just use the existing defaults